src/Http/Controllers/Admin/ModuleController.php

```diff
@@ -1698,8 +1700,16 @@ protected function getPermalinkBaseUrl()
         return $appUrl . '/'
             . ($this->moduleHas('translations') ? '{language}/' : '')
             . ($this->moduleHas('revisions') ? '{preview}/' : '')
-            . ($this->permalinkBase ?? $this->getModulePermalinkBase())
-            . (isset($this->permalinkBase) && empty($this->permalinkBase) ? '' : '/');
+            . (empty($this->getLocalizedPermalinkBase()) ? ($this->permalinkBase ?? $this->getModulePermalinkBase()) : '')
+            . (((isset($this->permalinkBase) && empty($this->permalinkBase)) || !empty($this->getLocalizedPermalinkBase())) ? '' : '/');
```

Comment:
Can this be updated to avoid having to provide the final `/` in `getLocalizedPermalinkBase()`? For example:
```
    protected function getLocalizedPermalinkBase()
    {
        return [
            'en' => 'content',
            'fr' => 'contenu',
        ];
    }
```
This would be more similar to the behavior of `$permalinkBase`.

Reply:
@pboivin . I've pushed an update to factor this in. The trailing slash is added on the Frontent end
```
      permalink: function () {
        const localizedPermalinkbase = this.localizedPermalinkbase.length > 0 ? JSON.parse(this.localizedPermalinkbase) : {}
        return Object.keys(localizedPermalinkbase).length > 0 ? ((this.currentLocale.value in localizedPermalinkbase) ? localizedPermalinkbase[this.currentLocale.value].concat('/',this.fieldValueByName('slug')[this.currentLocale.value]) : this.fieldValueByName('slug')[this.currentLocale.value] ): this.fieldValueByName('slug')[this.currentLocale.value]
      },
```

Due to how the Frontend Works by replacing parts of the baseUrl with the current locale;
```
      fullUrl: function () {
        return this.customlink || this.baseUrl
          .replace('{language}', this.currentLocale.value)
          .replace('{preview}/', this.published ? '' : 'admin-preview/') + this.permalink
      },
```
The section referenced in the initial comment serves to not include the permalinkBase `$this->permalinkBase` as part of the base URL when `getLocalizedPermalinkBase` is not empty.

```
    protected function getPermalinkBaseUrl()
    {
        $appUrl = Config::get('app.url');

        if (blank(parse_url($appUrl)['scheme'] ?? null)) {
            $appUrl = $this->request->getScheme() . '://' . $appUrl;
        }

        return $appUrl . '/'
            . ($this->moduleHas('translations') ? '{language}/' : '')
            . ($this->moduleHas('revisions') ? '{preview}/' : '')
            . (empty($this->getLocalizedPermalinkBase()) ? ($this->permalinkBase ?? $this->getModulePermalinkBase()) : '')
            . (((isset($this->permalinkBase) && empty($this->permalinkBase)) || !empty($this->getLocalizedPermalinkBase())) ? '' : '/');
    }
```

Reply:
@MamlukiSn Thank you for taking the time to look into this! Your solution works perfectly.